picture-stealer/src/main/java/com/bobocode/picturestealer/service/CameraService.java

```diff
@@ -0,0 +1,44 @@
+package com.bobocode.picturestealer.service;
+
+import com.bobocode.picturestealer.dto.Camera;
+import com.bobocode.picturestealer.entity.CameraEntity;
+import com.bobocode.picturestealer.manager.CameraRepository;
+import jakarta.persistence.LockModeType;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.data.jpa.repository.Lock;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Propagation;
+import org.springframework.transaction.annotation.Transactional;
+
+/**
+ * @author "Maksym Oliinyk"
+ */
+@Slf4j
+@Service
+@RequiredArgsConstructor
+public class CameraService {
+
+    private final CameraRepository cameraRepository;
+
+    @Lock(LockModeType.OPTIMISTIC_FORCE_INCREMENT)
```

Comment:
In my opinion, there is no need to lock anything in this case. I mean that I don't expect any UPDATE statements in this stealer app.

Reply:
@tboychuk Right now, there really aren't any updates, but that's it for now

Reply:
Currently deleted